File: ePrijevozSarajevo/ePrijevozSarajevo.Model/UserRole.cs
namespace ePrijevozSarajevo.Model
{
    public class UserRole
    {
        public int UserRoleId { get; set; }
        public int UserId { get; set; }
        public int RoleId { get; set; }
        public virtual Role Role { get; set; } = null!;
    }
}

File: ePrijevozSarajevo/UI/eprijevoz_mobile/lib/models/userRole.dart
import 'package:json_annotation/json_annotation.dart';
part 'userRole.g.dart';

@JsonSerializable()
class UserRole {
  int? userRoleId;
  int? userId;
  int? roleId;

  UserRole({this.userRoleId, this.userId, this.roleId});

  factory UserRole.fromJson(Map<String, dynamic> json) =>
      _$UserRoleFromJson(json);

  Map<String, dynamic> toJson() => _$UserRoleToJson(this);
}
